Tests/oss-fuzz/test_fuzzers.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+import sys
+
+import fuzzers
+import pytest
+
+from PIL import Image
+
+if sys.platform.startswith("win32"):
+    pytest.skip("Fuzzer is linux only", allow_module_level=True)
```

Comment:
```suggestion
    pytest.skip("Fuzzer is macOS and Linux only", allow_module_level=True)
```

Reply:
Well, technically, it's linux only, but the required bits don't fail on osx because it includes the find command.